nodes/light-node.md

```diff
@@ -81,40 +81,24 @@ Refer to
 [the ports section of the celestia-node troubleshooting page](../../nodes/celestia-node-troubleshooting/#ports)
 for information on which ports are required to be open on your machine.
 
-To start the light node with a connection to a validator node's gRPC endpoint
-(which is usually exposed on port 9090):
+::: code-group
 
-```sh
-celestia light start --core.ip <URI> --p2p.network <network>
+```sh [Mainnet Beta]
+celestia light start --core.ip rpc.celestia.pops.one --p2p.network celestia
```

Comment:
You may also omit network flag as this one is default

Reply:
Yea I saw that. I included it for consistency w/ the other tabs but can remove if reviewers prefer